src/patterns/grids/content-grid/content-grid.scss

```diff
@@ -0,0 +1,92 @@
+@use "../../../shared-styles/normalize"; // Always used first
+@use "../../../shared-styles/grid.scss";
+@use "../../../shared-styles/media-query";
+@use "../../../shared-styles/space";
+
+.content-grid {
+  --primary-column-inline-size: #{grid.get-overall-inline-size(grid.$main-column-count)};
+
+  @include grid.base(grid.$max-inline-size);
+  &.page-grid__start {
```

Comment:
Should this class have underscores or hyphens?

Reply:
This is the flavour of BEM we use for class naming and is correct: hyphen separators for spaces in the block names, a double underscore separator between the block and the element. See https://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/